### ISSUE TITLE
False positive for `ComposeContentEmitterReturningValues`?

### ISSUE THREAD

**Issue by svenjacobs** [slackhq/compose-lints#419]
The following code triggers a false positive warning for `ComposeContentEmitterReturningValues`.
The code doesn't emit any content into the composition. It just returns a value.

```kotlin
@Composable
fun rememberInsetsController(): WindowInsetsControllerCompat? {
    val view = LocalView.current
    val window = remember { view.context.findActivity()?.window } ?: return null
    return remember { WindowCompat.getInsetsController(window, view) }
}
```

Also I think the XML configuration of the related [documentation](https://slackhq.github.io/compose-lints/rules/#do-not-emit-content-and-return-a-result) has a typo. Shouldn't it be `content-emitters` instead of `allowed-composition-locals`?

Version: `1.4.0`

**Comment by ahmedre** [comment]
This also happens with Circuit `Presenter<UiState>` composable functions.

**Comment by ZacSweers** [comment]
hm, does it go away if you don't use `LocalView`? I wonder if the composition local is what's triggering this

**Comment by ZacSweers** [comment]
Found the issue
